_posts/2024-01-23-developer-portal.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "What we learned building our Ultimate developer portal"
+date: 2024-01-23
+description: "GitHub Actions are central to our CI/CD system since we adopted them less than two years ago. We prioritized scalability and usability from day one. Before jumping into features and complex scripting, we planned the architecture with a focus on the developer experience when starting a new repository or integrating new features or fixes into existing workflows. We considered composite actions and reusable workflows - but only intra-repo (using the same workflow to deploy to dev, stage and prod on different scenarios). This post outlines our journey, the challenges we overcame, and our future outlook."
+categories: CI/CD pipeline
+preview_image: /assets/img/posts/reusable-workflows_preview.png
+author: "Damian Bojar & Amit Jakubowicz"
```

Comment:
That's not going to work like this without adjustments. The name needs to be an exact match to the blog_authors.yml (check Readme here, step one: https://github.com/ultimateai/blog/blob/main/README.md#add-a-blog-post-or-podcast) and currently its a string, not an array, so we can only have a single author. 
To change that, you'd need to adjust this to an array instead of a string and make the display components iterate over authors

Reply:
Please make @dbojarUltimate the author, I just added a couple of sentences